libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -828,6 +867,162 @@ void AP_UAVCAN::notify_state_send()
     _last_notify_state_ms = AP_HAL::native_millis();
 }
 
+void AP_UAVCAN::gnss_send()
```

Comment:
maybe break into separate fix2 and yaw send fns

Reply:
done